ts/index.ts

```diff
@@ -6,4 +6,6 @@ export * as composite from "./message_contents/composite.pb";
 export * as privateKey from "./message_contents/private_key.pb";
 export * as publicKey from "./message_contents/public_key.pb";
 export * as signature from "./message_contents/signature.pb";
+export * as ciphertext from "./message_contents/ciphertext.pb";
 export * as contact from "./message_contents/contact.pb";
+export * as invitation from "./message_contents/invitation.pb";
```

Comment:
@neekolas, does this mean that all these types will have to be referenced as `ciphertext.Ciphertext` and `invitation.Invitation`?  Is there a way to export these as a single namespace like we have in Go?

Reply:
You could do `export * from ...`, which would expose the entire namespace. Does open up the possibility of collisions. Still might make sense for the smaller modules.

Reply:
Hm, maybe not, we're mixing multiple packages here. I guess I'll leave it as is?

Reply:
I think it's fine as is. Will be consistent at least. Everything can be found with $package.$message, as opposed to some message types being exposed at the top level and some being namespaced.